pyaedt/edb_core/components.py

```diff
@@ -838,6 +833,12 @@ def create_port_on_component(
             False will take the closest reference pin and generate one port per signal pin.
         refnet : string or list of string.
             list of the reference net.
+        port_name : string
+            Port name for overwriting the default port naming convention,
+            which is ``[component][net][pin]``. The port name must be unique.
+            If a port is already with the specified name already exists, the
+            default naming convention is used so that port creation does
+            not fail.
```

Comment:
```suggestion
            Port name for overwriting the default port-naming convention,
            which is ``[component][net][pin]``. The port name must be unique.
            If a port with the specified name already exists, the
            default naming convention is used so that port creation does
            not fail.
```